shared/devices/device-revoke/index.js

```diff
@@ -71,33 +101,34 @@ const DeviceRevoke = (props: Props) => (
   />
 )
 
-const styleHeader = {
-  marginBottom: globalMargins.tiny,
-}
-
-const styleTLF = {
-  marginBottom: globalMargins.xtiny,
-}
-
-const styleName = {
-  color: globalColors.red,
-  fontStyle: 'italic',
-  marginTop: 4,
-  textDecorationLine: 'line-through',
-}
-
-const styleDevicesContainer = {
-  ...globalStyles.flexBoxColumn,
-  alignItems: 'flex-start',
-  alignSelf: 'center',
-  border: '1px solid ' + globalColors.black_05,
-  borderRadius: 4,
-  height: 162,
-  marginBottom: globalMargins.small,
-  marginTop: globalMargins.small,
-  overflowY: 'scroll',
-  padding: globalMargins.small,
-  width: 440,
-}
+const styles = styleSheetCreate({
+  header: {
+    marginBottom: globalMargins.tiny,
+  },
+  TLF: {
+    marginBottom: globalMargins.xtiny,
+  },
+  name: {
+    color: globalColors.red,
+    fontStyle: 'italic',
+    marginTop: 4,
+    textDecorationLine: 'line-through',
+  },
+  container: {
+    ...globalStyles.flexBoxColumn,
+    alignItems: 'flex-start',
+    alignSelf: 'center',
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: globalColors.black_05,
+    borderRadius: 4,
+    height: 162,
+    marginBottom: globalMargins.small,
+    marginTop: globalMargins.small,
+    // overflowY: 'scroll',
```

Comment:
Did you mean to remove this line?

Reply:
Nope! Good catch!

Reply:
Oh actually it looks like `overflowY` is not a available react native value and `overflow` has [some limitations](https://facebook.github.io/react-native/docs/layout-props.html#overflow). Anyone have any suggestions on how to get the type of scrolling behavior we get on desktop on mobile also then?

Reply:
just use a <ScrollView> from common adapters, you can't style overflow in RN like desktop